packages/screens/AdministrationDashboard/CurrentlyHighLightedProject.tsx

```diff
@@ -0,0 +1,291 @@
+import React, { useState } from "react";
+import {
+  FlatList,
+  Image,
+  ImageBackground,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  useWindowDimensions,
+} from "react-native";
+import {
+  Bars2Icon,
+  CheckBadgeIcon,
+  CheckCircleIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+  PlusIcon,
+  TrashIcon,
+} from "react-native-heroicons/outline";
```

Comment:
We have already all needed icon in `assets`, please check if it's missing you can extract that icon from Figma. We just try to avoid to have many source of icons

Reply:
Fixed, PR updated